lab_2_tokenize_by_bpe/main.py

```diff
@@ -163,11 +163,12 @@ def decode(
     decoding = ''
     for number in encoded_text:
         for key in inverted_vocabulary.keys():
```

Comment:
wait, why not just access the value directly by the key?

Reply:
Done, but I'm not sure that I understood correctly what should be done